test/voice-broadcast/components/molecules/VoiceBroadcastPlaybackBody-test.tsx

```diff
@@ -70,7 +71,7 @@ describe("VoiceBroadcastPlaybackBody", () => {
             });
 
             it("should stop the recording", () => {
-                expect(playback.getState()).toBe(VoiceBroadcastPlaybackState.Playing);
+                expect(playback.getState()).toBe(VoiceBroadcastPlaybackState.Stopped);
```

Comment:
I'm confused by this test case, if the button label is 'resume voice broadcast' shouldn't it play the broadcast? Is this working as intended?

Reply:
Yes. Looks super weird and the test is medium useful. Changing this to test this calls `toggle`.